app/Http/Controllers/UserController.php

```diff
@@ -21,6 +22,7 @@ public function __construct()
         $this->middleware('getUserRole', ['only' => 'listUsers', 'listPolls', 'listPoll']);
```

Comment:
you middle ware name is "getUserRole" but actually it check is user is admin or not i think.

so name should be proper like "isAdmin" etc so its clear what the middleware does